src/table/components/head/ColumnAdjuster.tsx

```diff
@@ -77,12 +74,14 @@ const ColumnAdjuster = ({ column, isLastColumn, onColumnResize }: AdjusterProps)
       updateWidth(adjustedWidth);
     } else if (event.key === KeyCodes.SPACE || event.key === KeyCodes.ENTER) {
       preventDefaultBehavior(event);
-      confirmWidth();
       focusHeadMenuButton(event);
+
+      confirmWidth();
     } else if (event.key === KeyCodes.ESC) {
+      preventDefaultBehavior(event);
```

Comment:
what if we call this function before these if ... else cases? 
now you are calling it in all 3 possible cases

Reply:
yes but we don't want to call it for all keyboard events